scamp/scamp-3.c

```diff
@@ -1244,9 +1245,9 @@ void proc_100hz (uint a1, uint a2)
                 }
             }
         }
-      else if (netinit_tick_counter >= 3)
+      else if (netinit_biff_tick_counter >= 20)
```

Comment:
I missed this before my comment.  I hadn't been getting any issues, but wondered if this improved things.  Happy to revert it as no real change is seen, but then 20 is only a 200ms wait (which in human terms is nothing).